### ISSUE TITLE
Not handling ambiguous replies properly

### ISSUE THREAD

**Issue by huntc** [sbt/sbt-conductr#97]
It appears that the http "ambiguous" status code is not handled properly:

```
[error] (*:conduct) com.fasterxml.jackson.core.JsonParseException: Unrecognized token 'Specified': was expecting ('true', 'false' or 'null')
[error]  at [Source: java.io.StringReader@2ec32cae; line: 1, column: 10]
```

**Comment by huntc** [comment]
@markusjura - can you please have a look at this given that you're going to be in there?

**Comment by markusjura** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;2aYWyvJb&#x2F;121-tactical-activities"><img src="https:&#x2F;&#x2F;github.trello.services&#x2F;images&#x2F;trello-icon.png" width="12" height="12"> Tactical Activities</a>

**Comment by markusjura** [comment]
Fixed with PR #111.
